Wikipedia/Code/WMFWelcomeIntroductionViewController.swift

```diff
@@ -3,22 +3,21 @@ class WMFWelcomeIntroductionViewController: UIViewController {
     
     @IBOutlet fileprivate var titleLabel:UILabel!
     @IBOutlet fileprivate var subTitleLabel:UILabel!
-    @IBOutlet fileprivate var tellMeMoreButton:UIButton!
     @IBOutlet fileprivate var nextButton:UIButton!
     
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = UIColor.clear
         titleLabel.text = WMFLocalizedString("welcome-explore-new-ways-title", value:"New ways to explore", comment:"Title for welcome screens including explanation of new notification features").uppercased(with: Locale.current)
-        subTitleLabel.text = WMFLocalizedString("welcome-explore-new-ways-sub-title", value:"New notifications and widgets to help you discover interesting articles", comment:"Sub-title for exploration welcome screen including explanation of new notification features")
-        tellMeMoreButton.setTitle(WMFLocalizedString("welcome-explore-tell-me-more", value:"Tell me more", comment:"Title for link which when tapped shows explaination of new exploration features"), for: UIControlState())
+        subTitleLabel.text = WMFLocalizedString("welcome-explore-features-sub-title", value:"Discover landmarks near you or search around the world with the Places tab\n\nTravel back in time with On this day to learn what happened today in history", comment:"Sub-title for exploration welcome screen including explanation of new features.")
```

Comment:
If we want to be sure translation for 'Places' and 'On this day' here match its use elsewhere we'll probably want to substitute the respective values in here.